prime-angular-frontend/src/app/modules/health-auth-site-reg/pages/site-information-page/site-information-page.component.html

```diff
@@ -29,7 +29,9 @@
             <mat-label>Site ID/PEC Code</mat-label>
             <input matInput
                    [formControl]="formState.pec">
-            <mat-error>Required</mat-error>
```

Comment:
I don't see this in the ticket, but is this field not required anymore?

Reply:
It was never used